### ISSUE TITLE
UI glitched after running on Wayland, then on x11

### ISSUE THREAD

**Issue by fireclouu** [flathub/org.yuzu_emu.yuzu#1393]
### yuzu Version

1680

### Description

I'm using PopOs 22.04

Tried running yuzu on Wayland, it works fine, but I can't import some games since it fails telling that GPU is outdated (which does not occur on x11)

so I again switched to X11 then this happened:
![image](https://github.com/flathub/org.yuzu_emu.yuzu/assets/22563129/aa22268f-fa10-4970-907a-2f49bc48dc34)

![image](https://github.com/flathub/org.yuzu_emu.yuzu/assets/22563129/ca40c746-93fa-4976-9570-a3403b1bc350)

![image](https://github.com/flathub/org.yuzu_emu.yuzu/assets/22563129/d98e159f-feb1-48f8-86da-fa2caa83dbf8)

Even in-game status bar does not appear because of it (I toggle ctrl + s multiple times on this screenshot, and only gives me blank bars on bottom as you can see)
![image](https://github.com/flathub/org.yuzu_emu.yuzu/assets/22563129/6e902a8b-58df-4119-a774-cf8754bd4610)

I tried reinstalling yuzu via flatpak and AppImage, also clean install by removing all folders and files on my pc related to yuzu and it still persist on like that.

TLDR: Wayland breaks yuzu UI on X11

### System Information

```bash session
System:
  Host: ... Kernel: 6.6.6-76060606-generic x86_64 bits: 64 compiler: N/A
    Desktop: GNOME 42.5 Distro: Pop!_OS 22.04 LTS base: Ubuntu 22.04 LTS Jammy
Machine:
  Type: Laptop System: Acer product: Nitro AN515-58 v: V2.10
    serial: <superuser required>
  Mobo: ADL model: Jimny_ADH v: V2.10 serial: <superuser required>
    UEFI: Insyde v: 2.10 date: 07/07/2023
Battery:
  ID-1: BAT1 charge: 47.9 Wh (97.8%) condition: 49.0/57.5 Wh (85.2%)
    volts: 17.5 min: 15.4 model: LGC AP18E8M status: Charging
CPU:
  Info: 12-core (4-mt/8-st) model: 12th Gen Intel Core i5-12500H bits: 64
    type: MST AMCP arch: Alder Lake rev: 3 cache: L1: 1.1 MiB L2: 9 MiB
    L3: 18 MiB
  Speed (MHz): avg: 2315 high: 3966 min/max: 400/4500:3300 cores: 1: 3666
    2: 400 3: 989 4: 400 5: 3700 6: 400 7: 3734 8: 3966 9: 3056 10: 3289
    11: 3128 12: 2695 13: 1535 14: 2770 15: 2919 16: 400 bogomips: 99532
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx
Graphics:
  Device-1: Intel Alder Lake-P Integrated Graphics
    vendor: Acer Incorporated ALI driver: i915 v: kernel bus-ID: 0000:00:02.0
  Device-2: NVIDIA GA107M [GeForce RTX 3050 Ti Mobile]
    vendor: Acer Incorporated ALI driver: nvidia v: 545.29.06
    bus-ID: 0000:01:00.0
  Device-3: Chicony ACER HD User Facing type: USB driver: uvcvideo
    bus-ID: 1-6:2
  Display: x11 server: X.Org v: 1.21.1.4 driver: X:
    loaded: modesetting,nvidia unloaded: fbdev,nouveau,vesa gpu: i915
    resolution: 1920x1080
  OpenGL: renderer: Mesa Intel Graphics (ADL GT2)
    v: 4.6 Mesa 23.3.2-1pop0~1704238321~22.04~36f1d0e direct render: Yes
Network:
  Device-1: Intel Alder Lake-P PCH CNVi WiFi vendor: Rivet Networks
    driver: iwlwifi v: kernel bus-ID: 0000:00:14.3
  IF: wlp0s20f3 state: up mac: ...
  Device-2: Realtek vendor: Acer Incorporated ALI driver: r8169 v: kernel
    port: 3000 bus-ID: 0000:2b:00.0
  IF: enp43s0 state: down mac: ...
  IF-ID-1: lxcbr0 state: down mac: ...
  IF-ID-2: virbr0 state: down mac: ...
Drives:
  Local Storage: total: 1.38 TiB used: 1.05 TiB (76.6%)
Info:
  Processes: 452 Uptime: 1h 52m Memory: 7.47 GiB used: 6.84 GiB (91.7%)
  Init: systemd runlevel: 5 Compilers: gcc: 11.4.0 Packages: 2958 Shell: Zsh
  v: 5.8.1 inxi: 3.3.13
```

### Logs

[yuzu_log.txt](https://github.com/flathub/org.yuzu_emu.yuzu/files/13931275/yuzu_log.txt)

### Remarks

logs are empty, its on UI not in-game
